net/simple-adblock/files/simple-adblock.init

```diff
@@ -487,13 +496,13 @@ process_url() {
 	if ! $dl_command "$1" $dl_flag "$R_TMP" 2>/dev/null || [ ! -s "$R_TMP" ]; then
 		output 1 "$_FAIL_"
 		output 2 "[DL] $type $label $__FAIL__\\n"
-		echo -e -n "Error: downloading '${1}'.\\n" >> "${jsonFile}.error"
+		printf "%b" "Error: downloading '${1}'.\\n" >> "$sharedMemoryError"
```

Comment:
Wouldn't it make more sense to replace $(1) with %s?

Reply:
Oh, thank you, this can actually be replaced by the flagless echo. I struggle with force-pushing updates for existing PRs, so would appreciate if this can be merged as is.